This patch forces go generate to use the installed goyacc, instead of trying to
rebuild it from git.

--- a/vm/compiler.go
+++ b/vm/compiler.go
@@ -2,7 +2,7 @@
 // This file is available under the Apache license.
 
 // Build the parser:
-//go:generate $GOPATH/bin/goyacc -v y.output -o parser.go -p mtail parser.y
+//go:generate goyacc -v y.output -o parser.go -p mtail parser.y
 
 package vm
 
